en/docs/guides/identity-verification/identity-verification-overview.md

```diff
@@ -11,18 +11,15 @@ themselves from potential threats and enhance the overall security of their syst
 
 The WSO2 Identity Server facilitates identity verification by establishing a connection with a third-party provider through an identity verification IS connector. Within the WSO2 Identity Server, the user attribute's identity verification status is efficiently managed, along with the associated metadata from the identity verification provider.
 
-You can use the [identity verification management API](../../apis/idv-rest-api.md) to add, update and delete identity verification providers
+You can use the [identity verification management API]({{base_path}}/apis/idv-rest-api.md) to add, update and delete identity verification providers
 
 ## Configuring an Identity Verification Provider
 
-An identity verification provider (IDVP) is a third-party service that is used to verify the identity of a user. 
-Through Identity Server, you can configure an IDVP to verify the identity of a user. In order to onboard IDVP connectors,
-you have to deploy the required IDVP connectors from the connector store.
-
-In order to onboard an IDVP to WSO2 Identity Server, you need to deploy the identity verification connector artifacts.
```

Comment:
Removed this section, as we are not providing any information on how to download it from the connector store. Hope this is okay.